src/schema/objects/columns.yaml

```diff
@@ -112,6 +112,49 @@ duration:
     - type: string
       enum:
         - n/a
+eye1_x_coordinate:
+  name: eye1_x_coordinate
+  display_name: Eye 1 x-coordinate
+  description: |
+    Gaze position x-coordinate of the first recorded eye.
+  type: integer
+eye1_y_coordinate:
+  name: eye1_y_coordinate
+  display_name: Eye 1 y-coordinate
+  description: |
+    Gaze position y-coordinate of the first recorded eye.
+  type: integer
+eye1_pupil_size:
+  name: eye1_pupil_size
+  display_name: Eye 1 pupil size
+  description: |
+    Pupil size of the first recorded eye.
+  type: integer
+eye2_x_coordinate:
+  name: eye2_x_coordinate
+  display_name: Eye 2 x-coordinate
+  description: |
+    Gaze position x-coordinate of the second recorded eye.
+  type: integer
+eye2_y_coordinate:
+  name: eye2_y_coordinate
+  display_name: Eye 2 y-coordinate
+  description: |
+    Gaze position y-coordinate of the second recorded eye.
+  type: integer
+eye2_pupil_size:
+  name: eye2_pupil_size
+  display_name: Eye 2 pupil size
+  description: |
+    Pupil size of the second recorded eye.
+  type: integer
+eye_sample:
+  name: eye_sample
+  display_name: Eye-tracking sample onset
+  description: |
+    Onset of the sampled recorded eye(s) gaze position coordinates
+    and/or pupil size. According to the sampling scheme of the eye-tracker.
```

Comment:
I find this description confusing. Can you elaborate on what different sampling schemes might entail? Also, is it the onset, or just a given position at every time point? For example, if the eye position and pupil size stay exactly the same for 5 seconds, and you sample every 1 second, will there be 5 rows, or just one row?

Reply:
Yes ok, it wasn't really sample the right term, I replaced it by `eye_timestamp` and changed the description as it is simply a timestamp. To answer your question, in the case your eyes or pupil will stay exactly the same for 5 seconds (in theory impossible with noise of the recorder and human capacity) you would have 5 rows at 1 Hz of identical values with 5 different timestamps, giving something like [1133.0, 1134.0, 1135.0, 1136.0, 1137.0].

Reply:
Thanks for the clarification! Switching it to time stamp makes it much clearer.